packages/plugin-vue-jsx/src/index.ts

```diff
@@ -321,4 +340,8 @@ function getHash(text: string) {
   return hash('sha256', text, 'hex').substring(0, 8)
 }
 
+function ensureArray<T>(value: T | readonly T[]): readonly T[] {
+  return Array.isArray(value) ? value : ([value] as readonly T[])
+}
```

Comment:
To handle https://github.com/microsoft/TypeScript/issues/17002, the typings are a bit complicated.